packages/sveltekit/src/vite/sourceMaps.ts

```diff
@@ -251,6 +243,86 @@ export async function makeCustomSentryVitePlugins(options?: CustomSentryVitePlug
   return [...restOfSentryVitePlugins, customPlugin];
 }
 
+/**
+ * Whether the user enabled (true, 'hidden', 'inline') or disabled (false) source maps
+ */
+export type UserSourceMapSetting = 'enabled' | 'disabled' | 'unset' | undefined;
+
+/** There are 3 ways to set up source maps (https://github.com/getsentry/sentry-javascript/issues/13993)
+ *
+ *     1. User explicitly disabled source maps
+ *       - keep this setting (emit a warning that errors won't be unminified in Sentry)
+ *       - We won't upload anything
+ *
+ *     2. Users enabled source map generation (true, 'hidden', 'inline').
+ *       - keep this setting (don't do anything - like deletion - besides uploading)
+ *
+ *     3. Users didn't set source maps generation
+ *       - we enable 'hidden' source maps generation
+ *       - configure `filesToDeleteAfterUpload` to delete all .map files (we emit a log about this)
+ *
+ * --> only exported for testing
+ */
+export function changeViteSourceMapSettings(
+  viteConfig: {
+    build?: {
+      sourcemap?: boolean | 'inline' | 'hidden';
+    };
+  },
+  sentryPluginOptions?: CustomSentryVitePluginOptions,
+): UserSourceMapSetting {
```

Comment:
I see that we don't use this return value other than in tests. We'll need the return value for when we can automatically configure `filesToDeleteAfterUpload`, right?